bodyfetcher.py

```diff
@@ -208,10 +241,37 @@ def make_api_call_for_site(self, site):
                 owner_rep = 0
             q_id = str(post["question_id"])
 
-            is_spam, reason, why = check_if_spam(title, body, owner_name, owner_link, site, q_id, False, False, owner_rep, post_score)
+            is_spam, reason, why = check_if_spam(
```

Comment:
Quite true, but as of now, there's no improvement in reading ability -- if anything, it makes it slower to comprehend what's going on (as you now have to read 12 lines instead of 1).